lua/tests/infoview/autoopen_false_spec.lua

```diff
@@ -30,5 +30,18 @@ describe('infoview', function()
     assert.windows.are(lean_window)
     infoview.open()
     assert.windows.are(lean_window, infoview.get_current_infoview().window)
+    infoview.close()
+  end)
+
+  it('shows infoview contents immediately after they are opened', function(_)
+    vim.cmd('edit! ' .. fixtures.lean_project.path .. '/Test.lean')
+    assert.windows.are(lean_window)
+    infoview.open()
+    helpers.move_cursor{ to = {3, 27} }
```

Comment:
```suggestion
    helpers.move_cursor{ to = {3, 27} }
    infoview.open()
```

The other order used to work already.

Reply:
Thanks, good catch! Though that worried me to double check, and the test passes even without the change still, so I made some other error in it. Will look in a sec.

Reply:
OK now it fails without and passes with. I honestly don't know why what I did now is different than before :/ but there's some digging that needs doing later.